README.md

```diff
@@ -54,6 +54,15 @@ The mutt-wizard is run with the command `mw`. Once everything is setup, you'll u
 - `-f` -- Assume mailbox names and force account configuration without connecting online at all.
 - `-o` -- Configure mutt for an account, but do not keep mail offline.
 
+## Update Mailbox with Cron
+
+Since mutt-wizard requires shell variables stored in `~/.profile`, you will need to add `. $HOME/.profile` to your cronjobs. For example, to have mail update every 10 minutes, this is the line that should appear in the crontab file:
```

Comment:
Sorry I would object here, since not everone defines their environment in `~/.profile`. There are plenty of other possible places like

- `{/etc/bash,~/}.bashrc`
- `/etc/environment`
- `/etc/profile` / `/etc/profile.d/*`,

asf., or even non-default places like some random file somewhere in `$HOME` which is to be sourced at user login/startup.

**EDIT/** Maybe instead of `~/.profile` just reference something like `<configfile>` and explain that this means the file where relevant environment is setup. ?